rust/processor/src/utils/database.rs

```diff
@@ -146,6 +147,29 @@ pub async fn get_connection(pool: &PgPool) -> Result<AsyncConnectionWrapper<Asyn
 }
 */
 
+/// Gets the connection.
+/// If it was unable to do so (default timeout: 30s), it will keep retrying until it can.
+pub async fn get_conn(db_pool: &PgDbPool) -> PgPoolConnection {
```

Comment:
what about the other get_conn?

Reply:
still same question. I thought we had a get_conn in the trait. Are we not using it anymore in favor of this one?

Reply:
I delete this now